dept44-starter/src/main/java/se/sundsvall/dept44/configuration/WebConfiguration.java

```diff
@@ -72,6 +75,15 @@ FilterRegistrationBean<DisableBrowserCacheFilter> disableBrowserCacheFilterRegis
 		return registration;
 	}
 
+	@Bean
+	@ConditionalOnProperty(name = "mdc.municipalityId.enabled", havingValue = "true")
+	FilterRegistrationBean<MunicipalityIdFilter> municipalityIdFilterRegistration() {
+		final var registration = new FilterRegistrationBean<>(new MunicipalityIdFilter(municipalityIdLocation));
```

Comment:
municipalityIdLocation can be null here.

(Should this variable be defined as an `int` instead of an `Integer`, in order to avoid this? In the MunicipalityIdFilter the variable is defined as an int)

Reply:
municipalityIdLocation is set to 1 by default in constructor. Defined av Integer to be used with @Value